lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -35,31 +35,12 @@
 
 # We manually import all the classes which have a 'GetSchemaAttributeNames'
 # method so we have access to it and the 'pythonClass' method.
-from pxr import Usd, UsdGeom, UsdLux, UsdRender, UsdRi, UsdShade, UsdSkel, UsdUI, UsdVol, Kind, Tf
+from pxr import Usd, UsdGeom, UsdLux, UsdRender, UsdRi, UsdShade, UsdSkel, UsdUI, UsdVol, Kind, Tf, Sdr
 
 nameTxt = 'nameTxt'
 attrValueFld = 'attrValueFld'
 attrTypeFld = 'attrTypeFld'
 
-def getPrettyName(name):
```

Comment:
Moved to C++ function `UsdMayaUtil::prettifyName()` exported to Python as `mayaUsdLib.Util.prettifyName()`